doc/modules/grid_search.rst

```diff
@@ -741,3 +741,5 @@ This is currently implemented in the following classes:
     ensemble.ExtraTreesRegressor
     ensemble.GradientBoostingClassifier
     ensemble.GradientBoostingRegressor
+    ensemble.HistGradientBoostingClassifier
+    ensemble.HistGradientBoostingRegressor
```

Comment:
`HistGradientBoosting*`  does not have out of bag estimates. It splits the data once in the beginning into a validation set and uses that for early stopping. This is different from the paragraph above:

> This left out portion can be used to estimate the generalization error without having to rely on a separate validation set.

`HistGradientBoosting*` also uses the same training set for every iteration, which is different from the paragraph above:

> For each classifier in the ensemble, a different part of the training set is left out.

Reply:
Sure. But the discussion is about using a fraction of data for estimating the generalization error. `This left out portion can be used to estimate the generalization error without having to rely on a separate validation set. This estimate comes “for free” as no additional data is needed and can be used for model selection.` That's why I added them.

Reply:
I missed the Bootstrapping  discussion.  I was only thinking about generalization error. Sorry.

Reply:
I still do not see `HistGradientBoosting*` belonging in the list here. The paragraph above explicit states "... without having to rely on separate validation set", while `HistGradientBoosting*` explicitly uses a separate validation set.